MAUI/Cartesian-Charts/Axis/Types.md

```diff
@@ -334,6 +335,192 @@ chart.XAxes.Add(primaryAxis);
+
+### Customizing the range
+
+To customize the range of the log axis, you can use the [Minimum](), [Maximum]() and [Interval]() properties of the [LogarithmicAxis](). By default, an excellent range will be calculated automatically based on the provided data. The following code example demonstrates the range customization of the axis.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart.YAxes>
+
+    <chart:LogarithmicAxis Minimum="100" Maximum="10000"/>
+
+</chart:SfCartesianChart.YAxes>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+chart.YAxes.Add(new LogarithmicAxis() { 
+
+	Minimum = 100, 
+	Maximum = 10000 
+
+});
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis range customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_range.jpg)
+
+### Logarithmic base
+
+The log axis base can be customized using the [LogarithmicBase]() property of the axis. By default, the base of a logarithmic axis is set to 10. You can, however, set the base to any value that meets your requirements. Just ensure that the base value is any positive number greater than 1 and cannot be a decimal number. The following code example demonstrates the logarithmic values in the y-axis calculated from base 2.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart.YAxes>
+
+    <chart:LogarithmicAxis LogarithmicBase="2"/>
+
+</chart:SfCartesianChart.YAxes>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+chart.YAxes.Add(new LogarithmicAxis( ) { LogarithmicBase = 2 });
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis base customization in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_base.jpg)
+
+### Logarithmic interval
+
+The logarithmic interval can be customized using the [Interval]() property of the axis. In the logarithmic axis, the interval is based on powers. When the logarithmic base is 10 and the logarithmic interval is 1, then the axis labels are placed at an interval of 10. i.e., 1, 10, 100, 1000, etc. When the logarithmic interval is 2, then the axis labels are placed at an interval of 10^2. i.e., 1, 100, 1000, etc. The default value of the interval is 1, and it cannot be a decimal number.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart.YAxes>
+
+    <chart:LogarithmicAxis Interval="2"/>
+
+</chart:SfCartesianChart.YAxes>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+chart.YAxes.Add(new LogarithmicAxis( ) { Interval = 2 });
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![LogarithmicAxis interval in MAUI Chart](Axis_Images/maui_chart_logarithmic_axis_interval.jpg)
+
+### Actual minimum & Actual maximum
+
+The actual minimum property can be used to get the double value that represents the minimum value of the chart axis range and the actual maximum property can be used to get the double value that represents the maximum value of the chart axis range. The following code example demonstrates how to get the actual minimum and actual maximum values of the axis range using the labels.
```

Comment:
The ActualMinimum and ActualMaximum of the LogarithmicAxis used to get actual range of the axis. The following code example demonstrates how to get the actual minimum and maximum.

Reply:
@Saravanan-Madhesh ,Content modified as requested.